README.md

```diff
@@ -13,45 +13,106 @@ of taking a lot of time constructing the cluster and platform. We target:
-#### If you use Azure
-Use "az login" to log into your Azure cloud account at the machine.
-#### If you use GCP
-Set the `GOOGLE_APPLICATION_CREDENTIALS` environment variable as described in [the GCP docs](https://cloud.google.com/docs/authentication/getting-started).
+Replace `cloudtik[aws]` with `clouditk[azure]` or `cloudtik[gcp]` if you want to create clusters on Azure or GCP.
+Use `cloudtik[all]` if you want to manage clusters with all supported Cloud providers.
 
-### 4 Create a workspace for your clusters.
-CloudTik uses Workspace concept to manage your Cloud network and other resources. In a Workspace, you can start one or more clusters.
-Use the following command to create and provision a Workspace:
+You can install the latest CloudTik wheels via the following links. These daily releases do not go through the full release process.
+
+| Linux      | Installation                                                                                                                                       |
+|:-----------|:---------------------------------------------------------------------------------------------------------------------------------------------------|
+| Python 3.9 | `pip install -U "cloudtik[aws] @ https://d30257nes7d4fq.cloudfront.net/downloads/cloudtik/cloudtik-0.9.0-cp39-cp39-manylinux2014_x86_64.whl" `     |
+| Python 3.8 | `pip install -U "cloudtik[aws] @ https://d30257nes7d4fq.cloudfront.net/downloads/cloudtik/cloudtik-0.9.0-cp38-cp38-manylinux2014_x86_64.whl" `     |
+| Python 3.7 | `pip install -U "cloudtik[aws] @ https://d30257nes7d4fq.cloudfront.net/downloads/cloudtik/cloudtik-0.9.0-cp37-cp37m-manylinux2014_x86_64.whl" `    |
+
+
+### 3. Authentication to Cloud Providers API
+
+After CloudTik is installed on your working machine, then you need to configure or log into your Cloud account to 
+gain access to cloud provider API on this machine.
+then you need 
+
+#### AWS
+
+First, install AWS CLI(command line interface). Please refer to
+[AWS CLI Installation Guide](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) 
+for detailed instructions.
+
+After AWS CLI is installed, then you need configure AWS CLI about credentials. Please refer to 
+[Configuring the AWS CLI](https://github.com/aws/aws-cli/tree/v2#getting-started) for detailed instructions.
+
+The quickest way to configure is to run `aws configure` command as below, fill these items out then AWS CLI will 
+be configured and authenticated. *AWS Access Key ID* and *AWS Secret Access Key* can be found from the AWS guide of
+[managing access keys](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html).
 
 ```
-cloudtik workspace create your-workspace-config.yaml
+$ aws configure
+AWS Access Key ID [None]: ...
+AWS Secret Access Key [None]: ...
+Default region name [None]: ...
+Default output format [None]:
```

Comment:
This is too verbose. We don't need show these details. the above description as to aws-configure is enough.

Reply:
Moved this part to the next step.

Reply:
Deleted this detail.